operator/rpc_client.go

```diff
@@ -29,7 +30,7 @@ const (
 type RpcMessage = interface{}
 
 type AggregatorRpcClient struct {
-	rpcClientLock        sync.Mutex
+	rpcClientLock        sync.RWMutex
```

Comment:
Why do we need RW Mutex? Seems like just unnecessary overhead. This is not read oriented entity

Reply:
Well, the client needs to be read by whatever tasks need it, but needs to be written to when (re-)initializing, since it doesn't have a reconnection mechanism built-in. At the same time, the messaging rate is high, so it doesn't make sense to use a simple lock for both situations, we want to allow async message sending.

Reply:
Wait wait, we can't use read lock on sendRequest c.rpcClient.Call shall modify state internally